### PR TITLE
Datawave-MaxExapnsionRegexQueryTest-1023 

### PR DESCRIPTION
**Pull request by plainolneesh**
While testing for the periodic test failures (see #637) and (#1023), I found two tests that would fail because repetitive runs would hit the list limits.

### CODE REVIEW

**Review by ivakegg — APPROVED** [review]

**Review by apmoriarty — APPROVED** [review]

**Comment by ivakegg** [comment]
Looks like some of the files need to be run through the formatter first.  Do a full build (mvn clean install -Ddist -Pexamples) and then commit the formatting changes.

File: warehouse/query-core/src/test/java/datawave/query/MaxExpansionRegexQueryTest.java [hunk 1]
```diff
@@ -266,7 +266,10 @@ public void testMaxIvaratorResultsFailsQuery() throws Exception {
         runTest(query, expect);
         // verify that the ivarators ran and completed
         assertTrue(countComplete(dirs) >= 1);
```
Comment (ivakegg) [inline warehouse/query-core/src/test/java/datawave/query/MaxExpansionRegexQueryTest.java:268]: Don't need the assertTrue if the assertEquals is good....

File: warehouse/query-core/src/test/java/datawave/query/MaxExpansionRegexQueryTest.java [hunk 2]
```diff
@@ -309,6 +312,11 @@ public void testMaxIvaratorResults() throws Exception {
...
```
Comment (ivakegg) [inline warehouse/query-core/src/test/java/datawave/query/MaxExpansionRegexQueryTest.java:313]: Same comment as above.

**Review by ivakegg — CHANGES_REQUESTED** [review]

**Comment by ivakegg** [comment]
It appears there are formatting issues.  Please run the formatter and check in any changes (mvn -V -B -e clean formatter:format sortpom:sort -Pautoformat)

File: warehouse/query-core/src/test/java/datawave/query/MaxExpansionRegexQueryTest.java [hunk 3]
```diff
@@ -266,7 +266,9 @@ public void testMaxIvaratorResultsFailsQuery() throws Exception {
         runTest(query, expect);
         // verify that the ivarators ran and completed
         assertTrue(countComplete(dirs) >= 1);
-        
+
+        // clear list before new set is added
+        dirs.clear();
```
Comment (ivakegg) [inline warehouse/query-core/src/test/java/datawave/query/MaxExpansionRegexQueryTest.java:271]: How exactly is this supposed to help?  At line 273 we reassign the dirs variable so I am a little confused as to how this fixes the periodic failure.

**Review by ivakegg — CHANGES_REQUESTED** [review]

**Review by ivakegg — APPROVED** [review]

**Pull request merged**